### DISCUSSION TITLE
how do i put object in front of a character (instead of behind him like the background)

### DISCUSSION THREAD

**Discussion by Riskerman21** [KadeArchive/Kade-Engine#1608] [Q&A] [answered]
yeah how?
<img width="492" alt="Screen Shot 2021-08-06 at 1 09 34 pm" src="https://user-images.githubusercontent.com/61993263/128450237-276e43f0-b59e-4e57-9fa2-62ff6039cd94.png">

he is standing on the railing making it not look 3d lol.

**Comment by napstaa967** [comment, marked as answer]
fnf uses layering in order, aka if you add something then another line adding something else the second item will be rendered on top

go to the part after the camera positions inside playstate.hx and do this:

```hx
if (!PlayStateChangeables.Optimize)
		{
			add(gf);

			// Shitty layering but whatev it works LOL
			if (curStage == 'limo')
				add(limo);

			add(dad);
			add(boyfriend);
			if (curStage == 'your stage name')
				add(the item you wanna use on top);
		}
```
then go to the start and do this:
```hx
public var dialogue:Array<String> = ['dad:blah blah blah', 'bf:coolswag'];

	var halloweenBG:FlxSprite;
	var isHalloween:Bool = false;
	var the item name:FlxSprite;
```
